### PR TITLE
Models PR

### PR DESCRIPTION
**Pull request by MuhammadAmir5670**

### CODE REVIEW

File: detection/face_detector.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+CONF_TRESHOLD = 0.3
+IOU_THESHOLD = 0.5
+BASE_PATH = Path(os.path.dirname(os.path.abspath(__file__)))
+
+def get_coors(xywh, w, h):
+    x1 = int(xywh[0] * w - 0.5 * xywh[2] * w)
+    y1 = int(xywh[1] * h - 0.5 * xywh[3] * h)
+    x2 = int(xywh[0] * w + 0.5 * xywh[2] * w)
+    y2 = int(xywh[1] * h + 0.5 * xywh[3] * h)
+    return [x1, y1, x2, y2]
```
Comment (MuhammadAmir5670) [inline detection/face_detector.py]: Why do we need this function? I think the YoloV5 bounding box output format is already x1, y1, x2, and y2 just making an extra effort for nothing? please check accordingly and remove this function.

**Comment by MuhammadAmir5670** [comment]
Q) Do we need these files? if not please delete them accordingly. also check other files like these.
detection/data/hyps/hyp.Objects365.yaml
detection/data/hyps/hyp.VOC.yaml
detection/data/hyps/hyp.scratch-high.yaml
detection/data/hyps/hyp.scratch-low.yaml
detection/data/hyps/hyp.scratch-med.yaml

File: detection/distance_detector.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            if detections is not None and len(detections):
+                # Rescale boxes from img_size to im0 size
+                detections[:, :4] = scale_coords(image_.shape[2:], detections[:, :4], image.shape).round()
+
+                for *coors, conf, cls in detections:
+                    label = self.names[int(cls)]
+                    if label == 'person':
+                        results.append((label, coors))
+
+        return results
```
Comment (MuhammadAmir5670) [inline detection/distance_detector.py]: the following code is being repeated in the three files, please make it such that it does not repeat.
1) detection/face_detector.py
2) detection/distance_detector.py
3) detection/mask_detector.py

**Comment by MuhammadAmir5670** [comment]
Please look for all the .ymal and .sh files and delete them all if they are unnecessary? also, test the code after removing the files.

File: detection/distance_detector.py (hunk 2 above)
Comment (MuhammadAmir5670) [reply detection/distance_detector.py]: Reviews Done in [b333bd9](https://github.com/MuhammadAmir5670/intelligent-covid-SOP-compliance/pull/1/commits/b333bd972b0250753f031e29491cf0db2a105ce1) and [76b1304](https://github.com/MuhammadAmir5670/intelligent-covid-SOP-compliance/pull/1/commits/76b13040a4b36a906e298232437019c8a653987c)

File: detection/face_detector.py (hunk 1 above)
Comment (MuhammadAmir5670) [reply detection/face_detector.py]: Reviews Done in [b333bd9](https://github.com/MuhammadAmir5670/intelligent-covid-SOP-compliance/pull/1/commits/b333bd972b0250753f031e29491cf0db2a105ce1) and [76b1304](https://github.com/MuhammadAmir5670/intelligent-covid-SOP-compliance/pull/1/commits/76b13040a4b36a906e298232437019c8a653987c)

Comment (abdulmateen22) [reply detection/face_detector.py]: this function to for cropping the image actually it return n model gave the coordinates with txt for  this code convert to the image pixel

**Pull request merged**